0000-template.md

```diff
@@ -4,7 +4,7 @@
 * Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
 * Review Manager: TBD
 * Status: **Awaiting review**
-* Pull Request: [apple/swift#NNNNN](https://github.com/apple/swift/pull/NNNNN)
+* Implementation: [apple/swift#NNNNN](https://github.com/apple/swift/pull/NNNNN)
```

Comment:
How about **Awaiting implementation** instead of a PR link?  We are fine with pull requests being created before there is an implementation.

Reply:
Like this?

```markdown
* Implementation: **Awaiting implementation**
```

But if @krilnon needs to parse PR links in a required format, wouldn't it be better to have this in the template?

Reply:
I suspect @krilnon can adjust the parser to accept this format as well.  I think it is better than having a broken link — it conveys "action required".

@krilnon WDYT?

Reply:
As it stands, pull requests that haven't been merged in to `master` aren't parsed at all — because they haven't started a formal review and therefore don't show up on the status page. So the decision here doesn't matter much from the PoV of the parser. People will make slightly more formatting mistakes if we don't show the link format somewhere, but they can always look at past proposals rather than the template.

I do like the call to action implied by *Awaiting Implementation*. I think Ben's point is that the template file serves as an answer to the question "what structure do I need in my proposal document?", whereas the process.md file (and readme.md) answers the question "how do I go through the steps of proposing a change?".

Reply:
How about this?

```markdown
# Feature name

* Proposal: [SE-NNNN](NNNN-filename.md)
* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
* Review Manager: TBD
* Status: **Awaiting implementation**


*During the review process, add the following fields as needed:*

* Implementation: [apple/swift#NNNNN](https://github.com/apple/swift/pull/NNNNN)
* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution/), [Additional Commentary](https://lists.swift.org/pipermail/swift-evolution/)
* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/...commit-ID.../proposals/NNNN-filename.md)
* Previous Proposal: [SE-XXXX](XXXX-filename.md)
```

In the formulation above, **Awaiting Implementation** becomes a status, (or pseudo-status because it'll never be parsed), and the **Implementation** label is moved to the "add these list items as needed" section.

Reply:
```markdown
* Status: **Awaiting implementation**
```

I like this idea.

Reply:
Me too.